Makefile.dep

```diff
@@ -48,6 +48,10 @@ ifneq (,$(filter sixlowpan,$(USEMODULE)))
   USEMODULE += vtimer
 endif
 
+ifneq (,$(filter ng_ipv6_hdr,$(USEMODULE)))
+  USEMODULE += ng_inet_csum
```

Comment:
The idea is to remove this prefix as soon as the new stack is stable and the old one can be removed. We did this primarily to keep colisions with the old stack as low as possible. When the network stack is done, there will be a big rename commit and `ng_` will be a thing of the past ;-)